components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/IdentityMgtEventListener.java

```diff
@@ -325,9 +325,12 @@ public boolean doPostAuthenticate(String userName, boolean authenticated,
                         String emailTemplate = null;
                         int tenantId = userStoreManager.getTenantId();
                         String firstName = null;
+                        String userStoreDomain = userStoreManager.getRealmConfiguration().getUserStoreProperty(UserCoreConstants.RealmConfig.PROPERTY_DOMAIN_NAME);
```

Comment:
Too lengthy line, this should be divided from 120 length, please refer WSO2 code formatting guide.